<launch>


  <node name="ik_velocity_control" 
    pkg="interbotix_xsarm_control" 
    type="clik_velocity_control_node_armonly.py" 
    output="screen" 
    ns="mobile_wx250s">
  </node>

  <node name="velocity_comparison_plotter" 
    pkg="interbotix_xsarm_control" 
    type="plot_ee_tracking.py" 
    output="screen" 
    ns="mobile_wx250s">
  </node>
  
  <node name="compute_target_position_from_vicon" 
    pkg="interbotix_xsarm_control" 
    type="compute_target_position_from_vicon.py" 
    output="screen" 
    ns="mobile_wx250s">
  </node>
  
  <node name="ee_motion_planning" 
    pkg="interbotix_xsarm_control" 
    type="ee_motion_planning.py" 
    output="screen" 
    ns="mobile_wx250s">
  </node>



</launch>
